packages/as-proba/assembly/probability/binomial.ts

```diff
@@ -43,10 +43,13 @@ export class Binomial extends Sampler implements Drawer {
   }
 
   /**
-   * Draws a number.
+   * Draws a number from the binomial distribution using the rejection sampling method.
```

Comment:
No need to specify implementation in the title imo:

```suggestion
   * Draws a number from the binomial distribution
```